Set correct version number and macports tag

We don't want to use curl to fetch a version number and want to identify
our releases as being built in MacPorts to simplify origin tracking for
upstream in bug reports.

Upstream-Status: Inappropriate [configuration]
--- configure.orig	2016-11-04 00:10:51.000000000 +0100
+++ configure	2016-11-04 00:13:34.000000000 +0100
@@ -18,7 +18,7 @@
 # ========================================================
 
 name=TextMate
-ver=$(curl -sf "${rest_api}/releases/nightly/version")${tag:-+git.$(git rev-parse --short HEAD)}
+ver=@VERSION@${tag:-+macports.git.$(git rev-parse --short HEAD)}
 min_os=10.9
 
 # ===============================
